### ISSUE TITLE
images question

### ISSUE THREAD

**Issue by zhang92** [zizhaozhang/unet-tensorflow-keras#11]
python train.py
Found 0 images belonging to 0 classes.
Found 0 images belonging to 0 classes.
Found 0 images belonging to 0 classes.
Found 0 images belonging to 0 classes.
./datasets/
build UNet ...

please how are images and labels.
thank you very much
thank you very much...........

**Comment by li-pengcheng** [comment]
U can see this in loader...
